operator-common/src/test/java/io/strimzi/operator/common/operator/resource/AbstractNonNamespacedResourceOperatorIT.java

```diff
@@ -44,6 +49,11 @@ public static void before() {
         client = new DefaultKubernetesClient();
     }
 
+    @BeforeEach
+    public void renameResource() {
+        resourceName = getResourceName(RESOURCE_NAME);
+    }
+
```

Comment:
Why is this needed? Can you explain?

Reply:
I was running into some flakiness of the tests because all the resources had the same name.